.gitignore

```diff
@@ -282,3 +282,4 @@ samples/openapi3/client/petstore/go/privatekey.pem
 
 ## OCaml
 samples/client/petstore/ocaml/_build/
+/samples/client/jetbrains/adyen/checkout71/http/client/Apis/http-client.private.env.json
```

Comment:
can you please add `#jetbrains-http-client` (with a line break before that) as this line is not related to ocaml client generator at all?

Reply:
i'll do it after merging the PR. have a nice weekend

Reply:
done